dockstore-webservice/src/main/resources/swagger.yaml

```diff
@@ -5413,12 +5435,16 @@ paths:
         description: "repository path"
         required: true
         type: "string"
-      - name: "services"
```

Comment:
If I understand correctly, this would break backwards compatibility, but given low usage of permissions and services, it probably doesn't matter (also used only by UI and not CLI

Reply:
Yeah, any source code that uses the methods with the changed signatures will need to be adjusted/rebuilt.  Existing executables should continue to work fine.  These changes parrot an preexisting change at: https://github.com/dockstore/dockstore/blob/b602fd29f6a9fb4bbcd61970dbc149017d4710e4/dockstore-webservice/src/main/java/io/dockstore/webservice/resources/WorkflowResource.java#L858